Crash clicking "Permanent audiovisual mod" after deleting the mod in question.

**Platform:** Desktop
**Version:** Desktop
**Rulesets:** [LynxRo Tiles, Civ V - Vanilla, Civ style set by Bucketeer, Civ V - Gods & Kings, 5Hex-Tileset]
**Last Screen:** `com.unciv.ui.pickerscreens.ModManagementScreen`

--------------------------------

null

--------------------------------


**Message:**
```
java.lang.RuntimeException: Actor: CheckBox: Permanent audiovisual mod
	at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:192)
	at com.badlogic.gdx.scenes.scene2d.Actor.fire(Actor.java:152)
	at com.badlogic.gdx.scenes.scene2d.ui.Button.setChecked(Button.java:125)
	at com.badlogic.gdx.scenes.scene2d.ui.Button$1.clicked(Button.java:93)
	at com.badlogic.gdx.scenes.scene2d.utils.ClickListener.touchUp(ClickListener.java:88)
	at com.badlogic.gdx.scenes.scene2d.InputListener.handle(InputListener.java:71)
	at com.badlogic.gdx.scenes.scene2d.Stage.touchUp(Stage.java:355)
	at com.unciv.CrashHandlingStage.access$touchUp$s80204510(CrashHandlingStage.kt:10)
	at com.unciv.CrashHandlingStage$touchUp$1.invoke(CrashHandlingStage.kt:24)
	at com.unciv.CrashHandlingStage$touchUp$1.invoke(CrashHandlingStage.kt:24)
	at com.unciv.ui.utils.ExtensionFunctionsKt$wrapCrashHandling$1.invoke(ExtensionFunctions.kt:307)
	at com.unciv.CrashHandlingStage.touchUp(CrashHandlingStage.kt:24)
	at com.badlogic.gdx.InputEventQueue.drain(InputEventQueue.java:70)
	at com.badlogic.gdx.backends.lwjgl3.DefaultLwjgl3Input.update(DefaultLwjgl3Input.java:189)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:394)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:143)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:116)
	at com.unciv.app.desktop.DesktopLauncher.main(DesktopLauncher.kt:61)
Caused by: java.lang.NullPointerException
	at com.unciv.ui.pickerscreens.ModManagementScreen.refreshInstalledModActions(ModManagementScreen.kt:456)
	at com.unciv.ui.pickerscreens.ModManagementScreen.access$refreshInstalledModActions(ModManagementScreen.kt:32)
	at com.unciv.ui.pickerscreens.ModManagementScreen$refreshInstalledModActions$visualCheckBox$1.invoke(ModManagementScreen.kt:465)
	at com.unciv.ui.pickerscreens.ModManagementScreen$refreshInstalledModActions$visualCheckBox$1.invoke(ModManagementScreen.kt:458)
	at com.unciv.ui.utils.ExtensionFunctionsKt$toCheckBox$1$1.invoke(ExtensionFunctions.kt:214)
	at com.unciv.ui.utils.ExtensionFunctionsKt$toCheckBox$1$1.invoke(ExtensionFunctions.kt:213)
	at com.unciv.ui.utils.ExtensionFunctionsKt$onChange$1.changed(ExtensionFunctions.kt:75)
	at com.badlogic.gdx.scenes.scene2d.utils.ChangeListener.handle(ChangeListener.java:28)
	at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:188)
	... 17 more

```

**Save Mods:**
```
No mod data: kotlin.UninitializedPropertyAccessException: lateinit property gameInfo has not been initialized
```

**Save Data:**
<details><summary>Show Saved Game</summary>

```
No save data: kotlin.UninitializedPropertyAccessException: lateinit property gameInfo has not been initialized
```
</details>

--- a/core/src/com/unciv/ui/pickerscreens/ModManagementScreen.kt
+++ b/core/src/com/unciv/ui/pickerscreens/ModManagementScreen.kt
@@ -529,6 +529,7 @@ class ModManagementScreen(
                 question = "Are you SURE you want to delete this mod?",
                 action = {
                     deleteMod(mod.name)
+                    modActionTable.clear()
                     rightSideButton.setText("[${mod.name}] was deleted.".tr())
                 },
                 screen = this,

